package.json

```diff
@@ -57,16 +57,16 @@
     "start": "webpack-dev-server --config webpack.dev.config.js",
     "start-dev-profiling": "cross-env NODE_ENV=dev-profiling API_URL=https://daostack-alchemy.herokuapp.com S3_BUCKET=daostack-alchemy-staging webpack-dev-server --config webpack.dev.config.js",
     "start-docker": "webpack-dev-server --config webpack.docker.config.js",
-    "start-prod": "cross-env NODE_ENV=production webpack-dev-server --config webpack.prod.config.js",
+    "start-prod": "cross-env NODE_ENV=production webpack-dev-server --config webpack.dev.config.js",
```

Comment:
still not sure if we want this, seems like we aren't really testing the prod config ever locally then, but maybe thats ok